.github/pull_request_template.md

```diff
@@ -6,6 +6,12 @@ Closes [insert issue #]
 ## Solution
 <!-- How did you solve the problem? -->
 
+**Breaking Changes** 
+<!-- Does this PR contain any backward incompatible changes? If so, what are they and should there be special considerations for release? -->
+- [ ] Yes - this PR contains breaking changes
+- [ ] No - this PR is backwards compatible  
+- Details ...
```

Comment:
nit: extra line break here?

Reply:
oddly enough the formatting broke on preview on github, which is why i omitted it

**without extra line**
![Screenshot from 2021-05-18 12-00-26](https://user-images.githubusercontent.com/44049504/118588296-ab4d9100-b7d0-11eb-8a4f-ac14973a97ec.png)

**with extra line**
![Screenshot from 2021-05-18 12-00-55](https://user-images.githubusercontent.com/44049504/118588334-c4eed880-b7d0-11eb-85ad-434f06ef1dc0.png)

Reply:
Can move the details under the yes checkbox then

Reply:
fixed!